flow-typed/box-ui-elements.js

```diff
@@ -485,10 +489,10 @@ type Errors = {
 };
 
 type FileAccessStats = {
-    preview_count?: number,
-    download_count?: number,
-    comment_count?: number,
-    edit_count?: number,
+    preview_count: number,
```

Comment:
Part of this PR? Or the other one

Reply:
I'm honestly not sure what's going on here. When I rebased yesterday, it kept bumping this around. Will modify to see if it goes away